src/main/resources/scripts/announcements/generate_json.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+import argparse
+import json
+import mysql.connector
+import re
+
+# This script will fetch all the rows from the announcement table, then parse
+# the title out from the message content and return a json file with the newly
+# populated title and modified message content.
+#
+# Step #1: Run this script to generate a json file with the modified
+# announcement data.
+#
+# Step #2: Please review the json file to make sure the title was parsed
+# correctly from the message content.
+#
+# Step #3: To update the announcement table with the json file run the
+# update_announcement.py script next.
+
+def create_json_file(json_file, host, user, password, database):
+
+    json_data = []
+    json_file = open(json_file,"w+")
+
+    db = mysql.connector.connect(
+        host=host,
+        user=user,
+        password=password,
+        database=database
+    )
+    cursor = db.cursor()
+    cursor.execute("SELECT id, created_date, created_by_id, title, message, priority FROM announcement")
+    result = cursor.fetchall()
+    cursor.close()
+    db.close()
+
+    for id, created_date, created_by_id, title, message, priority in result:
+        
+        if not title:
```

Comment:
Kinda a funky oops here.  You select for `title`, but since we're automatically adding the fake title to the announcements in the liquibase it means that none of them get added to this file.  If you remove this `if` it works as expected.

Reply:
I forgot we changed the title to auto populate the title. I need to think about the downstream effects. I changed the if statement to check that the message is populated. As long as the message comes back as an empty string, it's not needed.

Reply:
That seems to work now.  Thanks